docker/mem0/main.py

```
@@ -0,0 +1,329 @@
+    user_id: Optional[str] = None
+    run_id: Optional[str] = None
+    agent_id: Optional[str] = None
+    filters: Optional[Dict[str, Any]] = None
+
+
+@app.post("/configure", summary="Configure Mem0")
+def set_config(
+    config: Dict[str, Any], _api_key: Optional[str] = Depends(verify_api_key)
+):
+    """Replace the running Memory instance with a new config."""
+    global MEMORY_INSTANCE
```

Ruff (0.15.6)

[warning] Using the global statement to update `MEMORY_INSTANCE` is discouraged

(PLW0603)